### DISCUSSION TITLE
Technical measures to make Mastodon safer for young people?

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#16578] [General]
I could imagine that it might make sense to include a function in Mastodon, but perhaps also in Fediverse services in general, if it is not already available, that makes it possible to hide the number of followers if necessary, so as not to become directly or indirectly dependent on a number of followers, as is often the case subliminally with Facebook, Instagram, Twitter, YouTube, Twitch and/or TikTok. To prevent this, I think such a function would be useful.

In concrete terms, this function could perhaps make sense if, for example, young creators from YouTube and/or Twitch migrate their presence on Facebook, Instagram, Twitter, YouTube, Twitch and/or TikTok to the Fediverse - gradually together with their viewers, mostly young people who probably know the Fediverse in the rarest cases, and to protect the young creators and the viewers?

What do you say?

**Comment by linux-lukas** [comment]
I could also imagine that various selectable mechanisms could be established in Mastodon, and perhaps in the Fediverse as a whole, which would help the instance administration to protect children and young people in particular from content in the Fediverse that is not intended for them (of a pornographic content, for example) - the German agreement on the Protection of Minors in the Media (JMStV) could perhaps be a good guide here.

**Comment by weex** [comment]
I'm interested to see these kinds of mechanisms be available. There's tremendous opportunity in the fediverse to experiment with additional ways to tweak UX toward healthier use.

**Reply by linux-lukas** [reply]
Pixelfed from Fediverse has implements these functions with one or two others.
![pixelfed](https://user-images.githubusercontent.com/47597136/130418458-6370d040-97e4-4856-a40b-c6c8975dba9c.png)

**Comment by ClearlyClaire** [comment]
Some Mastodon forks allow hiding follower counts. At that time, Gargron did not find it worth the additional code complexity. I don't know whether he changed his mind on this.

**Reply by linux-lukas** [reply]
Thank you for your contribution to this topic, @ClearlyClaire !

Considering that the services in the Fediverse are increasingly becoming good alternatives to Facebook, Instagram, Twitter, YouTube, Twitch, etc. (hereafter referred to as non-free social media), a larger (renewed?) discussion on whether it would make sense to introduce technical mechanism in the Fediverse services.

As shown above, the main developer of Pixelfed dansup has established various functions that could perhaps be adapted for other services in the Fediverse.
